platform/web/js/libs/library_godot_audio.js

```diff
@@ -453,6 +460,18 @@ class SampleNode {
 		const bus = GodotAudio.Bus.getBus(params.busIndex);
 		const sampleNodeBus = this.getSampleNodeBus(bus);
 		sampleNodeBus.setVolume(options.volume);
+
+		(async () => {
+			/** @type {AudioWorkletProcessor | null} */
+			this._positionWorklet = await this.createPositionWorklet(true);
+			if (this._positionWorklet && !this.isCanceled) {
+				this._source.connect(this._positionWorklet);
+				if (options.start) {
+					this.start();
+					this.isStarted = true;
+				}
+			}
+		})();
```

Comment:
Either wrap the code with a try/catch, or better, add a `.catch()` to the async call.